display/state.py

```diff
@@ -159,15 +159,13 @@ def playGame(screen, keyboard):
                         data = Projectile('data', random.randrange(-4, -8, -1), random.randrange(50, 100), random.randrange(1, 9), 5, 'data', False, human.collision_rect.x, human.collision_rect.y, data=personal_data)
                         sprites.add(data)
                         if personal_data == None:
-                            #zucc wins
-                            pass
+                            StateMachine.zucc_win(screen)
```

Comment:
This is not what the StateMachine should be doing. You need to return `StateMachine.ZUCC_WIN` and navigate to the static method from there